docs/global.css

```diff
@@ -55,58 +55,49 @@
   color: black;
 }
 
-:global body h1 {
-  color: colorsTextMain;
+:global h1,
+:global h2,
+:global h3,
+:global h4,
+:global h5 {
   font-family: "Poppins";
+  letter-spacing: -0.03em;
+  color: #2f2d44;
+}
+
+:global body h1 {
   font-weight: 500;
   letter-spacing: -0.03em;
   line-height: 58px;
   font-size: 31px;
-  margin: 0px;
```

Comment:
I think you forgot to put the `margin: 0` on the default styles.

Reply:
Didn't actually ever need this

Reply:
`h*` elements have a default margin which this overrides. Your docs may look weird with more margin. Your choice though 😄